### ISSUE TITLE
[WIP custom battle system] Wrong picture loading at loading save + event behaivour 

### ISSUE THREAD

**Issue by elsemieni** [EasyRPG/Player#2262]
#### Name of the game:
Technical demo of a custom menu and battle system from a unreleased game (La Leyenda Heroen - 2003, by @Soeufans ).
Download link (with author permission): https://drive.google.com/file/d/16RNJ1Bna3sEH-bERumFv4KerPI0adQ6w/view?usp=sharing

Note: It has lots of patches file and unrelated map files, but do not fear; the game does not uses it. 

#### Player platform:
Tested with lastest master Player (30-jun-2020).

#### Description
I wanted to make a more resumed bug report with test cases, but I think it will better idea upload the whole thing here, beacuse there's several interesting things to deal here. I will enumerate one each one.

#### Wrong picture loading

Steps:
-Start game.
-Press cancel to enter custom menu.
-Select Guardar (Save). Save in any slot.
-Select Cargar (Load), load the saved file. 

Expected behaivour (RPG_RT.exe):
![image](https://user-images.githubusercontent.com/1494687/86521505-b2b9b500-be1f-11ea-9dec-a5974d467ab3.png)

Current behaivour (Player master):
![image](https://user-images.githubusercontent.com/1494687/86521515-c49b5800-be1f-11ea-9703-839eadf28df9.png)

Note: It's not related about saving. RPG_RT can load both saves from itself and Player correctly, and Player loads saves from itself and RPG_RT incorrectly. 

#### Event behaivour
This is currently a mixture of several ones, I don't know if it's related to the quantity of events, or programming style of the author. 

Steps:
-Start game.
-Talk to the first pillar below.

Expected behaivour (RPG_RT, wrong one?)
<del>MIDI music doesn't play</del> -> EDIT: Fixed, not a bug.

Current behaivour (Player master)
<del>-MIDI music plays.</del> -> EDIT: Fixed, not a bug.
-Lots, and I mean LOTS of warnings in easyrpg.log, like an unoptimized engine or something like that... of these:

````
Debug: Image size out of bounds: System/Azul7 (160x80 < 160x640 < 160x80) [2679x] EDIT -> Non standard system graphic dimensions, explained below. 

Debug: Event 0 exceeded execution limit
````

The first one can be explained that the game uses a non-standard System resource, but it got that debug warning lots of times, I feel something is wrong. 
And the second one, Event 0 maybe came from a common event, but I can't relate who? (Or some sort of a strange/faulty event) ? 

I will try meanwhile to isolate these cases and bring more info If I have more about this case, but I think it will be a bit interesting at least. 

Grettings!

**Comment by carstene1ns** [comment]
The system graphic is used by most windows, backgrounds, texts etc. This explains why the message is counted so many times.
The real question is why do you need this big system graphics? The original engine cannot use it anyway.

**Comment by Soeufans** [comment]
Hello!

I'm the author if this beta and engine.

i don't mind about the default graphics and other default systems from RPGmaker2003 all i'm going to do, is all custom, even the text system, neither the comand "show text" with the V1.12a from 2k3,  i already made a game full customized, but its done in 2k9 ultimate, so its not compatible with EasyRPG

And my plan, is going to use and use the maximum capacity of 2K3 as i can from my engines.

This beta, took me around 1 month to complete and now it can be tested, and still i'm not done to finish the new version, so its going to take more time and more work, and to find some bugs and glitches for EasyRPG

**Comment by elsemieni** [comment]
Hello.
A little update. 

The music glitch is just a missing RTP resource (Player find it by RTP resource mapping meanwhile RPG_RT not), so that's not considered a bug, so removed from the main issue. 

I'm still a bit worried about ```Debug: Event 0 exceeded execution limit``` thing. Tracing it can be a bit tricky.

**Comment by Ghabry** [comment]
The picture loading issue is fixed by #2220 

The system warning could be disabled (or maybe only warn once?) because it is extremely spammy.
